x-pack/plugins/monitoring/server/lib/logstash/get_pipeline_vertex_stats_aggregation.ts

```diff
@@ -210,15 +237,16 @@ export function getPipelineVertexStatsAggregation(
 
   const config = req.server.config();
 
-  return fetchPipelineVertexTimeSeriesStats(
+  return fetchPipelineVertexTimeSeriesStats({
     query,
     logstashIndexPattern,
     pipelineId,
     version,
     vertexId,
     timeSeriesIntervalInSeconds,
-    config.get('monitoring.ui.max_bucket_size'),
+    // @ts-ignore not undefined, need to get correct config
```

Comment:
Wondering if it might be better to check the return and avoid the ts-ignore. Maybe something like this?


```typescript
  if (isNaN(maxBucketSize)) {
    return Promise.reject('monitoring.ui.max_bucket_size was not a number in kibana configuration');
  } else {
    return fetchPipelineVertexTimeSeriesStats({
      query,
      logstashIndexPattern,
      pipelineId,
      version,
      vertexId,
      timeSeriesIntervalInSeconds,
      maxBucketSize,
      callWithRequest,
      req,
    });
  }
```

Reply:
I was thinking the ts-ignore would be temporary until the config is correctly typed.  This is the current type: https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/server/types.ts#L131.  But it's incorrect because not all config values return string or undefined.    Many have default values and would not be undefined: https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/server/config.ts . I was thinking to create a separate issue.

Reply:
Yeah, it'd be nice if we could avoid string path lookup but that'd definitely be bigger work. Should we open up the issue now so we can include it in this comment then?